src/components/ChartContainer/D3ContainerUtils.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
My suspicion is that you got this warning when you had only one export. Now that you have three, it probably doesn't apply anymore and can be removed.

Reply:
Done.